tests/IB/explicit_ex0.cpp

```diff
@@ -42,10 +42,10 @@
 #include <ibtk/muParserRobinBcCoefs.h>
 
 // Set up application namespace declarations
-#include <ibamr/app_namespaces.h>
-
 #include <array>
 
+#include <ibamr/app_namespaces.h>
```

Comment:
Should we do a pass of `clang-format` before the next release?

Reply:
Yeah, definitely - it's in the release tasks document.

We should also set up another CI runner to error out if clang-format complains because things like this sometimes slip through, but I want to get the test suite running first.